plugin/mlsdbprovider.cpp

```diff
@@ -528,10 +539,7 @@ void MlsdbProvider::updatePositioningEnabled()
 
 void MlsdbProvider::cellularNetworkRegistrationChanged()
 {
-    if (m_positioningEnabled) {
-        qCDebug(lcGeoclueMlsdb) << "cellular network registrations changed, updating position";
-        calculatePositionAndEmitLocation();
-    }
+    m_signalUpdateCell = true;
```

Comment:
Is this also triggered in the case where the signal strength of different cells has changed, but the actual registrations has not (e.g., we are not seeing different cell tower signals than previously, but we are seeing changed signal strengths, which could be used to calculate offline position via triangulation)?

Actually, I am not against preventing an online request in the "signal strength changed" case anyway, since that is likely to occur often due to e.g. environmental changes anyway.  But just curious as to the precise intent here.

Reply:
Ah, I guess this isn't related to the online requests anyway.  In any case, it's still fine as-is, so long as we do re-calculate the position from the available signal strength data every 10 seconds (or whatever the appropriate timeout is).